src/main/domain/vendorlist/Language.js

```diff
@@ -0,0 +1,28 @@
+import {InvalidLanguageValueError} from './error/InvalidLanguageValueError'
+
+class Language {
+  constructor(languageValue) {
+    this._value = this._extractValue(languageValue)
+  }
+
+  get value() {
+    return this._value
+  }
+
+  _extractValue(languageValue) {
+    if (!languageValue || DEFAULT_LANGUAGE_VALUE === languageValue) {
+      return DEFAULT_LANGUAGE_VALUE
+    }
+    if (languageValue.toLowerCase() !== languageValue) {
+      throw new InvalidLanguageValueError(languageValue)
+    }
+    if (languageValue.length !== 2) {
+      throw new InvalidLanguageValueError(languageValue)
+    }
+    return languageValue
+  }
+}
+
+const DEFAULT_LANGUAGE_VALUE = 'en'
```

Comment:
maybe we should use as default spanish? Or is it a specification of IAB that the default is english?

Reply:
In the API side, the default is English and this is prepared in our backend to not call for any purpose-xx translation (in fact, the purposes-en.json does not exist, so language=en just returns the original vendorlist.json)